### PR TITLE
Trent dev

### PR DESCRIPTION
**Pull request by trestoh**

### CODE REVIEW

File: WasatchNET/SPISpectrometer.cs [hunk 1]
```diff
@@ -470,25 +483,44 @@ public bool writeEEPROM(List<byte[]> pages)
...
                 byte currPage = (byte)(0x80 | (uint)page);
 
                 transmitData = new byte[1] { currPage };
                 command = wrapCommand(EEPROM_OPS, transmitData, STANDARD_PADDING);
+                logger.hexdump(command, String.Format("commit to EEPROM command for page {0}: ", page));
                 result = spi.readWrite(command);
+
+                logger.hexdump(result, "response: ");
+
+                Thread.Sleep(2);
```
Comment (mzieg) [inline WasatchNET/SPISpectrometer.cs:523]: Is this actually required?

**Review by mzieg — APPROVED** [review]
Thanks for helping Gaj!

File: WasatchNET/SPISpectrometer.cs (hunk 1 above)
Comment (trestoh) [reply WasatchNET/SPISpectrometer.cs:523]: The sleep or the dump?

Comment (trestoh) [reply WasatchNET/SPISpectrometer.cs:523]: It looks like it's there for the loop around

**Pull request merged**